package.json

```
@@ -1,32 +1,29 @@
 {
   "name": "witnet-solidity",
-  "version": "1.1.7",
+  "version": "2.0.0",
   "description": "Witnet CLI import module for Solidity projects",
-  "author": "Guillermo Diaz <guillermo@otherplane.com>",
+  "author": "Witnet Foundation",
   "license": "MIT",
   "private": false,
+  "type": "commonjs",
   "bin": {
-    "witnet": "build/bin/witnet.js"
+    "witnet": "src/bin/witnet.js"
+  },
+  "exports": {
+    ".": "./witnet/assets/index.js",
+    "./assets": "./witnet/assets/index.js",
+    "./utils": "./witnet/scripts/utils.js"
   },
   "files": [
-    "assets/witnet/**",
-    "build/bin",
-    "contracts/WitnetMigrations.sol",
-    "migrations/**",
-    "test/**",
-    ".env_witnet",
-    "truffle-config.js"
+    "src",
+    "witnet",
+    ".env_witnet"
   ],
   "scripts": {
-    "build": "babel src -d build",
-    "compile": "npm run build && truffle compile",
-    "console": "truffle console",
-    "fmt": "eslint test --ext .js,.ts,.cjs --ignore-path .gitignore && solhint --max-warnings 0 \"contracts/**/*.sol\"",
-    "fmt!": "eslint test --ext .js,.ts,.cjs --fix --ignore-path .gitignore && solhint --max-warnings 0 \"contracts/**/*.sol\" --fix",
-    "migrate": "truffle migrate --network",
+    "fmt": "eslint test --ext .js,.ts,.cjs --ignore-path .gitignore && solhint --max-warnings 0 \"witnet/migrations/contracts/**/*.sol\"",
+    "fmt!": "eslint test --ext .js,.ts,.cjs --fix --ignore-path .gitignore && solhint --max-warnings 0 \"witnet/migrations/contracts/**/*.sol\" --fix",
```

This should make the linter work again
```suggestion
    "fmt": "eslint --ext .js,.ts,.cjs . --ignore-path .gitignore && solhint --max-warnings 0 \"witnet/migrations/contracts/**/*.sol\"",
    "fmt!": "eslint test --ext .js,.ts,.cjs . --fix --ignore-path .gitignore && solhint --max-warnings 0 \"witnet/migrations/contracts/**/*.sol\" --fix",
```